fixture-monkey-kotlin/src/main/kotlin/com/navercorp/fixturemonkey/kotlin/introspector/PrimaryConstructorArbitraryIntrospector.kt

```diff
@@ -82,7 +83,9 @@ class PrimaryConstructorArbitraryIntrospector : ArbitraryIntrospector {
     ): Any? {
         return try {
             val parameterKotlinType = parameter.type.jvmErasure
-            if (parameterKotlinType.isValue) {
+            if (arbitrariesByPropertyName[parameter.name] is Duration) {
+                arbitrariesByPropertyName[parameter.name]
+            } else if (parameterKotlinType.isValue) {
```

Comment:
I think it is a better approach to modify `KotlinPropertyCache#getMemberProperties` for `value class`.

Reply:
There's no case for constructor injection for the value class, because it's injected directly into the value.

So I don't think we need to do anything to inject constructors.

Reply:
It's not working the way we thought it would! We'll ask for a review again when we're done.

Reply:
I think it is a good approach to add a new implementation of `Property` for value class, such as`ValueClassProperty`. `KotlinPropertyCache#getMemberProperties` would generate a `ValueClassProperty` for value class.

In this approach, You do not need to modify a specific ArbitraryIntrospector, `PrimaryConstructorArbitraryIntrospector`

All you have to do is add a new ArbitraryIntrospector for `ValueClassProperty`.
Then, you do not need to an `if` statement below because a new ArbitraryIntrospector will generate a value class type  `CombinableArbitrary`.

```kotlin
  if (parameterKotlinType.isValue) {
                parameterKotlinType.primaryConstructor!!.isAccessible = true
                parameterKotlinType.primaryConstructor!!.call(arbitrariesByPropertyName[parameter.name])
            }
```

Reply:
I The reason written to handle value class in the PrimaryConstructorArbitraryIntrospector is because the actual type is different from the value returned as AnnotatedType in KPropertyProperty.

For value objects, I modified it to include the existing type and it worked fine.